core/src/ast/query.rs

```diff
@@ -70,6 +70,24 @@ pub enum TableFactor {
         alias: TableAlias,
         size: Expr,
     },
+    Dictionary {
+        dict: Dictionary,
+        alias: TableAlias,
+    },
+}
+
+#[derive(Debug, Clone, PartialEq, Eq, Hash, Serialize, Deserialize)]
+pub enum Dictionary {
+    GlueTables,
+    GlueTableColumns,
+}
+impl Dictionary {
+    pub fn get_name(&self) -> &'static str {
+        match self {
+            Dictionary::GlueTables => "GLUE_TABLES",
+            Dictionary::GlueTableColumns => "GLUE_TABLE_COLUMNS",
+        }
+    }
```

Comment:
How about to try `strum` which is being used well in [ast/data_type.rs](https://github.com/gluesql/gluesql/blob/main/core/src/ast/data_type.rs)?

Then,
```rust
use strum_macros::Display;

...

#[derive(Debug, Clone, PartialEq, Eq, Hash, Serialize, Deserialize, Display)]
#[strum(serialize_all = "SCREAMING_SNAKE_CASE")]
pub enum Dictionary {
    GlueTables,
    GlueTableColumns,
}
```